### ISSUE TITLE
HPC-GAP: IsomorphismGroups groups error and lockup

### ISSUE THREAD

**Issue by fingolfin** [gap-system/gap#1209]
In HPC-GAP, the following error currently occurs:
```
gap> IsomorphismGroups(Stabilizer(MathieuGroup(23),23), MathieuGroup(22));
Error, List Element: <list>[14] must have an assigned value in
   called from
you can 'return;' after assigning a value
img := img ^ S.transversal[i]; at GAPROOT/hpcgap/lib/stbc.gi:832StabChainSwap( S ) at GAPROOT/hpcgap/lib/stbc.gi:772 called from
StabChainForcePoint( S.stabilizer, pnt ) at GAPROOT/hpcgap/lib/stbc.gi:772 called from
StabChainForcePoint( S, new ) at GAPROOT/hpcgap/lib/stbc.gi:1100 called from
ChangeStabChain( R[d], [ rbase.base[d] ], false ); at GAPROOT/hpcgap/lib/stbcbckt.gi:1243 called from
PBEnumerate( d + 1, false ) at GAPROOT/hpcgap/lib/stbcbckt.gi:1409 called from
...  at stream:1
brk>
```

And if one enters the command a second time, HPC-GAP locks up, with Ctrl-C not working, so I have to `kill -9` it.

**Comment by fingolfin** [comment]
Now the first issue did not occur for me anymore -- GAP just locked up... :-/

**Comment by ChrisJefferson** [comment]
I don't think ctrl-c ever worked in HPCGAP, but hint, you can type `!break` to enter the break loop. After that, you can see that the same problem is occurring, somehow an invalid transversal is being built, where we wander around an infinite loop. I have no idea how that invalid transversal is appearing however...

**Comment by fingolfin** [comment]
This has been resolved by f9f4d659e38c2646f7e30da708d49e43f4857b4b, part of PR #1442. 

Background: `CopyStabChain` in `lib/stbc.gi` contains an evil hack using `MASTER_POINTER_NUMBER`, which wasn't working in HPC-GAP, yet when merging the `hpcgap-default` branch into `master`, I did not realize this, and thus replaced the HPC-GAP counterpart with this new code that could not work (w/o the above fix) in HPC-GAP. Note that the old code in GAP wasn't quite correct either. See issue #1468
